src/entrypoint.rs

```diff
@@ -1,4 +1,3 @@
-#![cfg(feature = "program")]
```

Comment:
This global attribute prevents the `entrypoint` module from being compiled unless the `--features program` is passed in the CLI